roles/spark/templates/spark-defaults.conf.j2

```diff
@@ -19,10 +19,17 @@
 # This is useful for setting default environmental settings.
 
 # Example:
-spark.master                     mesos://zk://master01:2181,master02:2181,master03:2181/mesos
-# spark.eventLog.enabled           true
-# spark.eventLog.dir               hdfs://namenode:8021/directory
-# spark.serializer                 org.apache.spark.serializer.KryoSerializer
+
+spark.master                     {{mesos_master}}
+
+spark.eventLog.enabled           true
+spark.eventLog.dir               hdfs:///shared/spark_logs
+
+spark.history.fs.logDirectory    hdfs:///shared/spark_logs
+
+# spark.serializer               org.apache.spark.serializer.KryoSerializer
+
 spark.driver.memory              5g
 spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
-spark.mesos.executor.home	 /usr/lib/spark/spark-1.6.0-bin-hadoop2.6
+
+spark.mesos.executor.home	       {{spark_home}}
```

Comment:
The issue is more that I'm not sure if derby files behave nicely on HDFS (I think it'd be weird if it did since it'll update the file often which is not recommended for data on HDFS)